jaeger-core/src/main/java/io/jaegertracing/internal/JaegerTracer.java

```diff
@@ -354,7 +360,7 @@ private JaegerSpanContext createChildContext() {
         }
       }
 
-      return new JaegerSpanContext(
+      return JaegerTracer.this.tracingFactory.createSpanContext(
```

Comment:
just curious, why fully qualified accessor instead of jus `tracingFactory.createSpanContext` ?  I think the latter is also visible.

Reply:
`JaegerTracer.SpanBuilder` is a non-static nested class. I am able to access the tracer's `tracingFactory` from the inner class. Personally, not a huge fan of non-static nested classes (prefer explicitly passing instance to static nested class instead).